crypto/fipsmodule/rand/rand.c

```diff
@@ -73,11 +73,12 @@
 #endif
 
 #if defined(FIPS_ENTROPY_SOURCE_JITTER_CPU)
-static const unsigned kReseedInterval = 16777216;
+  static const unsigned kReseedInterval = 16777216;
```

Comment:
nitpick: not your change, but the variable naming here is a bit confusing w/r/t NIST.SP.800-90Ar1, which refers to "Maximum number of requests between reseeds" as `reseed_interval`, i.e. what we call `kMaxReseedCount` elsewhere